tools/update-zuliprc-api-field

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+"""
+This script is for updating API field of `zuliprc` with
+correct keys from database. It is for local development only.
+Ensure running this outside of vagrant environment.
+Usage:
+./tools/update-zuliprc-api-field /path/to/zuliprc_dev
+"""
+
+import argparse
+import configparser
+import subprocess
+import os
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('path', metavar='FILE', type=str, nargs='+',
+                        help="config file of dummy users in development server")
+    args = parser.parse_args()
+
+    zuliprc = configparser.ConfigParser()
+    zuliprc_paths_list = args.path
+
+    for zuliprc_path in zuliprc_paths_list:
+        if not os.path.exists(zuliprc_path):
+            raise RuntimeError("No zuliprc found at path %s" % (zuliprc_path,))
+
+        try:
+            with open(zuliprc_path, 'r') as f:
+                zuliprc.read_file(f, zuliprc_path)
+
+            if 'api' in zuliprc:
+                api_details = zuliprc['api']
+                if "email" in api_details:
```

Comment:
Not a big deal, but maybe have parenthetical consistency?

Reply:
Yikes! Thanks for pointing this out @whoodes.